src/libs/json-ld.js

```diff
@@ -0,0 +1,116 @@
+import I18nText from 'shared_components/I18nText';
+
+export function getServiceJsonLdData(service, canonicalUrl) {
```

Comment:
This function makes a bit difficult to know which fields to expect on each type of service, maybe three different functions would be better (and maybe each of them calling another with the common fields).
It's a nice to have, if you are busy just merge this.

Reply:
Completely makes sense.
I've updated the code.